docs/concepts/events-and-resources.md

```diff
@@ -0,0 +1,104 @@
+---
+description: Events are the primary units of observability in Prefect Cloud
+tags:
+    - resources
+    - events
+    - observability
+---
+
+# Events
+
+Events are the primary unit of observability in Prefect Cloud, and are designed to be a simple but complete record of something occuring somewhere in your stack at a given time, and power several features in prefect cloud, including flow run logs, audit logs, and automations. Prefect events can represent API calls, state changes, or events from your execution environment or infrastructure. Events enable analysis of your data stack via [event feed](/ui/events/), and the confiuration of stack reactivity via [automations](/ui/automations/).
+
+![Prefect UI](../img/ui/event-feed.png)
+
+## Event Specificiation
+
+Events adhere to a structured [specification](https://app.prefect.cloud/api/docs#tag/Events) for consistency.
+
+![Prefect UI](../img/ui/event-spec.png)
+  
+| Name | Type | Required? | Description |
+| ---- | ---- | --------- | ----------- |
+| occurred |  String | yes | When the event happened from the sender's perspective |
```

Comment:
Would it be simpler to say that it is the time that the event was sent? Is that accurate?